.github/workflows/build.yaml

```diff
@@ -34,6 +34,17 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  cpp-checks:
```

Comment:
Now that I think about it, this job is probably better suited for the `test.yaml` workflow since it's not technically a build job.

In our nightly workflow, the `test.yaml` workflow is triggered after a successful `build.yaml` workflow completion.

So the consequence of including this in `build.yaml` is that it will prevent `test.yaml` from being triggered in our nightly pipelines if `cpp-checks` is the only failing job, which is probably not what we want.

Reply:
done